### DISCUSSION TITLE
教程-用户指南   中的  请求体-多个参数  章节中代码错误

### DISCUSSION THREAD

**Discussion by liufei1997** [fastapi/fastapi#8511] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Union

from fastapi import Body, FastAPI
from pydantic import BaseModel

app = FastAPI()


class Item(BaseModel):
    name: str
    description: Union[str, None] = None
    price: float
    tax: Union[float, None] = None


class User(BaseModel):
    username: str
    full_name: Union[str, None] = None


@app.put("/items/{item_id}")
async def update_item(
    *,
    item_id: int,
    item: Item,
    user: User,
    importance: int = Body(gt=0),
    q: Union[str, None] = None
):
    results = {"item_id": item_id, "item": item, "user": user, "importance": importance}
    if q:
        results.update({"q": q})
    return results
```

### Description

教程-用户指南   中的  请求体-多个参数  中的  多个请求体参数和查询参数   中的示例代码出现错误
https://fastapi.tiangolo.com/zh/tutorial/body-multiple-params/#_3
![image](https://user-images.githubusercontent.com/58357252/171589310-93241230-93e1-4a9c-811e-778e93dd4f42.png)

Body参数缺少一个位置参数，即TypeError: Body() missing 1 required positional argument: 'default'

个人建议改为 Body(..., gt=0),

### Operating System

Windows

### FastAPI Version

无

### Python Version

3.7

**Comment by unights** [comment, marked as answer]
Hi @liufei1997
Maybe you need to update the version of fastapi to [0.78.0](https://github.com/tiangolo/fastapi/releases/tag/0.78.0).
See https://github.com/tiangolo/fastapi/issues/4911#issuecomment-1127729335
